docs(nuxt): Add missing code snippets (#11501)
getsentry/sentry-docs: platform-includes/user-feedback/example-widget/javascript.nuxt.mdx

In Nuxt, the best place to collect user feedback on the **client side** is inside a plugin:

```javascript {tabTitle:TypeScript} {filename:plugins/report-errors.ts}
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.hook('vue:error', (error, instance, info) => {
    Sentry.showReportDialog({ /* optional options */});
  })
})
```

> To collect user feedback for errors on the **server side**, you can create an error page like described in the Nuxt docs on [error handling](https://nuxt.com/docs/getting-started/error-handling#nitro-server-errors).

Reviewer: ```suggestion
To collect user feedback for errors on the **server side**, you can create an error page by following the [error handling](https://nuxt.com/docs/getting-started/error-handling#nitro-server-errors) instructions in the Nuxt docs.
```

> 

Author: ```suggestion
```

> 
> 
> 

Reviewer 2: super-nit: do we need these empty lines?

```suggestion
```
